ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -56,14 +115,14 @@
             "Path to a PKCS#12 formatted key store; used to enable TLS on inbound connections.",
         arity = "1",
         required = true)
-    private File keystoreFile;
+    private File keyStoreFile;
 
     @Option(
         names = "--tls-keystore-password-file",
         description = "Path to a file containing the password used to decrypt the keystore.",
```

Comment:
description should contain "key store"

Reply:
changed description on line 115 to contain "keystore" rather than "key store" - is that what was intended?

Reply:
👍